services/app-api/handlers/otel_handler.ts

```diff
@@ -0,0 +1,23 @@
+import opentelemetry from "@opentelemetry/api";
+import { NodeTracerProvider } from "@opentelemetry/sdk-trace-node";
+import { Resource } from "@opentelemetry/resources";
+import { OTLPTraceExporter} from "@opentelemetry/exporter-trace-otlp-grpc"
+import { SimpleSpanProcessor, ConsoleSpanExporter } from "@opentelemetry/sdk-trace-base";
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+const provider = new NodeTracerProvider({
```

Comment:
I think this file belongs somewhere else. We should probably create a new module called “otel” that exports this tracer, rather than having it live with our lambda handlers